pdc/apps/utils/utils.py

```diff
@@ -59,3 +61,33 @@ def read_permission_for_all():
 
 def get_model_name_from_obj_or_cls(obj_or_cls):
     return ContentType.objects.get_for_model(obj_or_cls).model
+
+
+class RelatedNestedOrderingFilter(OrderingFilter):
+    """
+    Extends OrderingFilter to support ordering by fields in related models
+    using Django orm '__', this change is based on rest_framework 3.2.5
```

Comment:
In 3.2.5, the OrderingFilter just works for the fields in model, but it can't ordering by the nested fields. 
Like there are four values in model(ReleaseComponentRelationship) can be ordered: 
['relation_type', 'id', 'from_component', 'to_component']
from_component and to_component are foreign keys.
The values here:
"to_component": {
        "id": "int", 
        "name": "string", 
        "release": "Release.release_id"
    }, 
And as default if you ordered by from_component or to_component, you can just ordered by the id of them. And you can just call <server_url>/?orderging=to_component.

I wrote this class to let people can use such apis, like <server_url>/?orderging=to_component__name. or <server_url>/?orderging=to_component__release.